src/main/java/uk/gov/hmcts/reform/translate/model/Dictionary.java

```diff
@@ -1,18 +1,16 @@
 package uk.gov.hmcts.reform.translate.model;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import lombok.Getter;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
 
 import java.util.Map;
 
-@Getter
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
 public class Dictionary {
-
-    private final Map<String, String> translations;
```

Comment:
Removing  @JsonCreator will break some functionality .

Reply:
Could you elaborate please, what does it break?

Reply:
Previously  marshal and un-marshall was not working with plain settings.    After testing again with latest version it is working without @JsonCreator.   I can be removed .